src/site/markdown/develop/computes/decode.md

```diff
@@ -0,0 +1,24 @@
+keywords: decode, base64, string
+description: Use the "decode" compute operation to reverse an encoding (e.g., Base64) applied to a selected column.
+
+# `decode`
+
+The Decode compute reverses an encoding previously applied to the values in the selected column.
```

Comment:
Use the `Decode` compute to reverse the encoding previously applied to the selected column, such as retrieving the original text from Base64.

Reply:
The Encode step is not mandatory as the value may already be collected in Base64, so I would remove `previously` 
```markdown
Use the `Decode` compute to reverse the encoding applied to the selected column such as retrieving the original text from Base64.
```